### ISSUE TITLE
Gmagick for PHP 8.0 / 8.1 / 8.2 Docker images

### ISSUE THREAD

**Issue by cameronterry** [Automattic/vip-container-images#386]
On a project I've been involved in hosted on WordPress.com VIP, we've been trying to do work with `Gmagick` for some handling of image previews for PDFs. However, after setting up the environment using the VIP CLI, I noticed the `Gmagick` library was missing.

I've noticed in the Dockerfiles, the library is commented out except for PHP 7.4;

https://github.com/Automattic/vip-container-images/blob/dc8998c05c7ec87927ce5d93b62c7bca820db316/php-fpm/Dockerfile.80#L15

https://github.com/Automattic/vip-container-images/blob/dc8998c05c7ec87927ce5d93b62c7bca820db316/php-fpm/Dockerfile.81#L15

https://github.com/Automattic/vip-container-images/blob/dc8998c05c7ec87927ce5d93b62c7bca820db316/php-fpm/Dockerfile.82#L21

Would it be possible to have Gmagick included for the PHP 8 Docker image?

As noted in the [VIP Lobby post here](https://lobby.vip.wordpress.com/2022/07/06/working-together-the-path-to-php-8-0/), PHP 8 is the preferred version since November 2022 and as it is a relatively new project I am on, we are on PHP 8. So at the moment the discrepancy is causing a bit of extra work.

Thank you for your time and assistance.

**Comment by rinatkhaziev** [comment]
Hey @cameronterry, thanks for bringing that up, we've added 8.0 and 8.1 extensions in the PR above. 

Does 7.4 work as expected for you?

**Comment by cameronterry** [comment]
Hey @rinatkhaziev - thank you for the quick response and PR. PHP 7.4 was working with `Gmagick` as expected when I switched over using the `vip dev-env update --php=7.4` command.

**Comment by cameronterry** [comment]
Another update - `Gmagick` is available with the PHP 7.4 container, but it appears when I use a PDF file with it I get this error;
```
Fatal error: Uncaught Exception: Postscript delegate failed (/wp/wp-content/uploads/2022/12/test.pdf)
```
From what I can ascertain, putting a `phpinfo(INFO_ENVIRONMENT);` somewhere and checking the output, is that Ghostscript is missing. Thinking back to a helpdesk ticket I created a while ago (asking about Imagick/Gmagick/missing PDF previews), that *might* not be an issue for this ...

Is Ghostscript available on VIP servers?
